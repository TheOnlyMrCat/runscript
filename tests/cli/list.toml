args = ["-l"]
fs.cwd = "in"
