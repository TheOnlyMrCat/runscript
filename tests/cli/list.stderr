[0m[1mTargets from `[0m[1m[38;5;14m.run[0m[1m`:
[0m[0m[33malpha [0m[31m.[0m[1m[38;5;12mR[0m[32m.[0m
bravo [0m[31m.[0m[34m.[0m[32m.[0m[1m[38;5;15m clean[0m

[0m[1mTargets from `[0m[1m[38;5;14mrun[0m[1m`:
[0m[0m[33malpha   [0m[31m.[0m[34m.[0m[1m[38;5;10mT[0m
charlie [0m[1m[38;5;9mB[0m[1m[38;5;12mR[0m[32m.[0m
delta   [0m[31m.[0m[1m[38;5;12mR[0m[32m.[0m[1m[38;5;15m alt[0m[1m[38;5;15m alt2[0m
...
